python/lsst/pipe/tasks/imageDifference.py

```diff
@@ -65,6 +65,9 @@ class ImageDifferenceConfig(pexConfig.Config):
         doc="Use a simple gaussian PSF model for pre-convolution (else use fit PSF)? "
             "Ignored if doPreConvolve false.")
     doDetection = pexConfig.Field(dtype=bool, default=True, doc="Detect sources?")
+    doDecorrelation = pexConfig.Field(dtype=bool, default=False,
+        doc="Perform diffim decorrelation correction similar to Kaiser? If True, also updates the "
```

Comment:
"similar to Kaiser" is not very informative. A more formal reference would be very helpful. As a very picky point, I suggest "update" instead of "updates" for consistency with the first sentence.